Components/landingPage.css

```diff
@@ -0,0 +1,63 @@
+.main-slider-container{
+    background-image:url('/Assests/Book-Landing-Image.png');
+    width:100%;
+    height:25rem;
+    background-repeat: no-repeat;
+    margin-top: 2rem;
+    display: flex;
```

Comment:
write helper class for flex properties so you don't have to write everytime flex properties